public/video-ui/src/components/Videos/AdvancedActions.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {getStore} from '../../util/storeAccessor';
+
+class AdvancedActions extends React.Component {
+    // the permissions are also validated on the server-side for each request
+    permissions = getStore().getState().config.permissions;
+    showActions = this.permissions.deleteAtom;
+
+    renderDelete() {
+        if(!this.permissions.deleteAtom) {
+            return false;
+        }
+
+        const doDelete = () => {
+            this.props.videoActions.deleteVideo(this.props.video);
+        };
+
+        return <li className="action-list__item">
+            <button className="btn label__expired action-list__button" onClick={doDelete}>DELETE</button>
```

Comment:
As deletion is un-recoverable, should we add a confirmation step to ensure its not clicked by accident? e.g. a popup that requires you to enter the atom id or something?